test/run_test.py

```diff
@@ -64,13 +64,11 @@
     'function_schema',
 ]
 
-# skip < 3.3 because mock is added in 3.3 and is used in rpc_fork and rpc_spawn
+# skip < 3.3 because mock is added in 3.3 and is used in rpc_spawn
 # skip python2 for rpc and dist_autograd tests that do not support python2
 if PY33:
     TESTS.extend([
-        'rpc_fork',
         'rpc_spawn',
```

Comment:
since we only have spawn mode on the OSS side, do we still need to explicitly have a "spawn" in the file name?

Reply:
we can keep it for now? once we completely get rid of fork, we can merge rpcSpawnTest with RpcTest